### PR TITLE
scraper fixes for MN, WY, DC

### PR DESCRIPTION
**Pull request by smcclure17**
- the DC demographic scraper was failing because demographic ID's (age,race,ethnicity,sex) combinations were missing from `covid_demographics.csv`, causing the scraper to fail on insert. This adds the necessary demographic variables.
 - Updates the MN url (we have to fetch this manually because the page is blocked by a capthca).
 - Updates some logic for extracting the dose data from the WY HTML.

### CODE REVIEW

**Review by mikelehen — APPROVED** [review]
LGTM with naming nits.

File: can_tools/scrapers/official/WY/WYVaccines.py [hunk 1]
```diff
@@ -40,8 +38,12 @@ def fetch(self):
         }
         return requests.get(self.source, headers=header).text
 
-    def _extract_regex(self, match, data):
-        end = re.findall(f"{match}: <strong>(.*?)</strong>", data)[0]
+    def _extract_data(self, match, data):
+        # use regex to find the tags that follow the name the doses (eg: "Total Doses: <strong>...data...</strong>")
```
[the remark this answers is not in the document]
Comment (smcclure17) [reply can_tools/scrapers/official/WY/WYVaccines.py:42]: oh, yeah, that doesn't make any sense haha. I'll fix + make more clear

File: can_tools/scrapers/official/WY/WYVaccines.py [hunk 2]
```diff
@@ -40,8 +38,12 @@ def fetch(self):
...
+        end = re.findall(f"{match}:\s?<strong>(.*?)</strong>", data)[0]
+
+        # convert back to an HTML object to extract the text of the tag containing the actual data
+        end = bs(end, "html.parser").find_all('span')[0].text
```
[the remark this answers is not in the document]
Comment (smcclure17) [reply can_tools/scrapers/official/WY/WYVaccines.py]: good point, I'll update

File: can_tools/scrapers/official/WY/WYVaccines.py [hunk 3]
```diff
@@ -40,8 +38,12 @@ def fetch(self):
         }
         return requests.get(self.source, headers=header).text
 
-    def _extract_regex(self, match, data):
-        end = re.findall(f"{match}: <strong>(.*?)</strong>", data)[0]
+    def _extract_data(self, match, data):
```
[the remark this answers is not in the document]
Comment (smcclure17) [reply can_tools/scrapers/official/WY/WYVaccines.py:41]: both of those sgtm!

thanks for the feedback 🙏

**Pull request merged**